### DISCUSSION TITLE
Leaf Documentation Ideas

### DISCUSSION THREAD

**Discussion by mychidarko** [leafsphp/leaf#244] [Ideas]
Hi community!

It's that time of the year again, the docs aren't living up to expectation. There've been complaints and the team has also noticed some issues like difficulty navigating the documentation. How can we make the documentation experience better for you? Drop your ideas down in the comments.

Cheers!

**Comment by mychidarko** [comment]
Documentation @ https://leafphp.dev

**Reply by phederal** [reply]
**TailwindCSS install with step-by-step instruction**

_case:_ when I first got acquainted with leaf, I had quite a few difficulties getting tailwincss to work with the framework

**Reply by mychidarko** [reply]
Noted. Will be addressing this issue really soon

**Comment by phederal** [comment]
**Restructure docs**
The documentation needs to be restructured and a complete refactoring needs to be done. Making it easier for newcomers who are just diving into Leaf. At the same time, the description of each section is kept detailed so that advanced Leaf users can quickly find the information they need about Leaf methods and modules.

**Reply by mychidarko** [reply]
I agree. I've started putting some ideas together. I'll push what I have during the week for more comments 🤞🏽

**Comment by terrybr** [comment]
The Leaf's InertiaJS documentation section needs to be updated.  The steps aren't working anymore. Unrelated to the documentation, the Leaf's InertiaJS implementation needs to also be updated since there are discrepancies with current packages versions and Leaf's CLI commands.
